### DISCUSSION TITLE
Helperline Feature - deactivate edge helper lines and only activate center helper lines

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#1250] [Q&A] [answered]
After a lot of testing with the new helper line feature (which is really great) I came to a point where I recognize that for complex BPMN models it is easier align elements to the center-line only and not the additional edge helper lines?

Is there any way to deactivate the edge helper lines?

**Comment by martin-fleck-at** [comment, marked as answer]
Yeah, showing all lines may be overkill for certain use cases, so we made the helper line types configurable through the `IHelperLineOptions`. You probably need something like this.

```ts
container.bind<IHelperLineOptions>(TYPES.IHelperLineOptions).toConstantValue({
    elementLines: [HelperLineType.Center, HelperLineType.Middle], // only show center and middle lines
    viewportLines: [] // do not show alignment lines for viewport
  });
```

**Reply by rsoika** [reply]
Hi @martin-fleck-at , yes this works and improves my situation a lot!
